src/config_editorconfig.ts

```diff
@@ -10,7 +10,7 @@ const getEditorConfig = memoize((folderPath: string, filesNames: string[]): Prom
   for (let i = 0, l = filesNames.length; i < l; i++) {
     const fileName = filesNames[i];
     const filePath = fastJoinedPath(folderPath, fileName);
-    if (!Known.hasFilePath(filePath)) continue;
+    // if (!Known.hasFilePath(filePath)) continue;
```

Comment:
Commented out for now, should probably be controlled by a flag since we don't populate `Known` when formatting from `stdin`.